src/test/java/io/jenkins/plugins/oidc_provider/KeysTest.java

```diff
@@ -105,33 +118,52 @@ public class KeysTest {
         JSONObject key = keys.getJSONObject(0);
         assertEquals("global", key.getString("kid"));
     }
-    @SuppressWarnings({"deprecation", "rawtypes"})
```

Comment:
You lost this.

Reply:
<img width="1097" height="258" alt="image" src="https://github.com/user-attachments/assets/0e964665-02bd-464a-9332-51062c6ba8d4" />

🤷🏼‍♂️